src/components/comment/CommentForm.jsx

```diff
@@ -62,9 +66,11 @@ const CommentForm = (props) => {
                 className="comment-form-control"
                 name="comment"
                 as="textarea"
-                placeholder={intl.formatMessage({
-                  id: "comment.form.placeholder",
-                })}
+                placeholder={
+                  intl.locale !== "en"
```

Comment:
ufff ... this is not very nice, can we put it to a function? and call the function instead of having this?